### DISCUSSION TITLE
Different Channels for Different Projects

### DISCUSSION THREAD

**Discussion by benb98** [ryanrosello-og/playwright-slack-report#206] [Q&A]
Is there anyway to have the channel where the message is sent be different dependant on the Playwright project being ran? For example I have multiple projects and multiple channels (let's call them X, Y and Z). I can't see anyway I can run X tests and have a message sent to channel X without sending a message to channel Y an Z as well.

**Comment by ryanrosello-og** [comment]
maybe try adding a ternary condition in your config?
e.g.
```
      {
        channels: process.env.THE_PROJECT==='Y' ? ["pw-tests", "ci"] : ["someotherchannel"], // provide one or more Slack channels
        sendResults: "always", // "always" , "on-failure", "off"
      },
```
